cincinnati/build.rs

```diff
@@ -0,0 +1,13 @@
+extern crate protoc_rust;
+
+use std::env;
+
+fn main() {
+    println!("{}/{}", env::var("OUT_DIR").unwrap(),"src/");
```

Comment:
I've tried your suggestion and it can be made to work. An annoyance is that features aren't passed to member crates from virtual crates. We're working around this in [dist/pr_check.sh](https://github.com/openshift/cincinnati/blob/master/dist/pr_check.sh#L16-L22).

Since your suggestion is to not enable this on CI we wouldn't have to go through that for the regular builds but we'll have to at least document it.